apps/loggregator.go

```diff
@@ -67,7 +67,12 @@ var _ = AppsDescribe("loggregator", func() {
 		})
 
 		It("exercises basic loggregator behavior", func() {
-			Eventually(logs, (Config.DefaultTimeoutDuration() + time.Minute)).Should(Say("(Connected, tailing|Retrieving) logs for app"))
+			if Config.GetUseLogCache() {
+				// log cache cli will not emit header unless being run in terminal
+				Consistently(logs, (Config.DefaultTimeoutDuration() + time.Minute)).ShouldNot(Say("(Connected, tailing|Retrieving) logs for app"))
```

Comment:
Hey @johannasmith. I think we could probably do without this assertion. Since it's a negative assertion inside of a `Consistently`, the assertion is almost always going to run through each poll for the entire timeout; it will take `DefaultTimeoutDuration` plus 1 minute to run. Since there's [a good assertion below](https://github.com/cloudfoundry/cf-acceptance-tests/pull/304/files#diff-9043d6eb59a3fedf9989ec85b9b04cd5R81), maybe just have the guard clause do nothing when Log Cache is being used:
```
if !Config.GetUseLogCache() {
  Eventually(logs, ...)
}
```

Reply:
@dsabeti Sounds good. We pushed the change you suggested. We're re-running the tests in our CI and will ping you when they go green.

Reply:
@dsabeti they passed. good to pull in this change